### PR TITLE
MapLookupFactory need to be Ser/Desr ready.

### PR DESCRIPTION
**Pull request by b-slim**

### CODE REVIEW

File: processing/src/test/java/io/druid/query/lookup/LookupSnapshotTakerTest.java [hunk 1]
```diff
@@ -113,44 +109,4 @@ public void testLookupPullingFromNonExistingFile() throws IOException
     List<LookupBean> actualList = lookupSnapshotTaker.pullExistingSnapshot();
     Assert.assertEquals(Collections.EMPTY_LIST, actualList);
   }
-
-  @JsonTypeName("mock")
-  private class LookupFactoryMock implements LookupExtractorFactory
```
Comment (drcrallen) [inline processing/src/test/java/io/druid/query/lookup/LookupSnapshotTakerTest.java]: Can we get rid of the other mock instance as well?

Comment (b-slim) [inline processing/src/test/java/io/druid/query/lookup/LookupSnapshotTakerTest.java]: @drcrallen done

**Comment by drcrallen** [comment]
:+1: after travis

**Comment by nishantmonu51** [comment]
:+1:

**Pull request merged**